src/main/webapp/js/maps-page-loader.js

```diff
@@ -7,6 +7,11 @@ const MAP_NIGHT_TYPE_ID = 'night'
  * New modes/map styles can be added in extraMapStyles.json file.
  *
  */
+
+let map;
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
```

Comment:
Maybe the confusion could be caused by the word edit, which is a verb and a noun, so it is unclear if this is an action, or a marker for an edit.

I'm not sure what's a better word, so perhaps explaining the context in the comments more might help.

Another possible reason for the confusion could be that editMarker is defined pretty far from where it is used (because it is a global). Not using a global is definitely going to be a lot more work, but if you can find an easy way to avoid it, and the `map` above, that would be good. First thing that comes to mind would be to use a class.